AzureIotPnpDps.X/main.c

```diff
@@ -7,582 +7,1322 @@
 #include "platform/application_manager.h"
 #include "platform/system.h"
 #include "platform/led.h"
+#include "platform/pin_manager.h"
 #include "platform/sensors_handling.h"
 #include "platform/cloud/cloud_service.h"
+#include "platform/cryptoauthlib/lib/atca_command.h"
+#include "platform/cryptoauthlib/lib/basic/atca_basic.h"
+#include "platform/drivers/timeout.h"
+#include "platform/delay.h"
 #include "platform/debug_print.h"
 #include "platform/mqtt/mqtt_core/mqtt_core.h"
 #include "platform/mqtt/mqtt_packetTransfer_interface.h"
 #include "azure/core/az_span.h"
 #include "azure/core/az_json.h"
-#include "azure/iot/az_iot_pnp_client.h"
-
-#define DEFAULT_START_TEMP_CELSIUS 22
-
-#define RETURN_ERR_WITH_MESSAGE_IF_FAILED(ret, msg)   \
-  do {                                                \
-    if (az_result_failed(ret)) {                       \
-        debug_printError(msg);                        \
-        return rc;                                    \
-    }                                                 \
-  } while (0)
-
-#define RETURN_WITH_MESSAGE_IF_FAILED(ret, msg)     \
-  do {                                              \
-    if (az_result_failed(ret)) {                     \
-        debug_printError(msg);                      \
-        return;                                     \
-    }                                               \
-  } while (0)
-
-extern az_iot_pnp_client pnp_client;
+#include "azure/iot/az_iot_hub_client.h"
+#include "main.h"
+
+extern az_iot_hub_client hub_client;
 extern pf_MQTT_CLIENT pf_mqtt_iotprovisioning_client;
-extern pf_MQTT_CLIENT pf_mqtt_pnp_client;
+extern pf_MQTT_CLIENT pf_mqtt_iothub_client;
```

Comment:
It looks like your changes are reverting a previous PR.

https://github.com/Azure-Samples/Microchip-PIC-IoT-Wx/pull/14